lib/hooks/core/hook-http.js

```diff
@@ -63,7 +63,7 @@ function requestWrap(request) {
     var labels = {};
     labels[TraceLabels.HTTP_METHOD_LABEL_KEY] = options.method;
     labels[TraceLabels.HTTP_URL_LABEL_KEY] = uri;
-    var span = agent.startSpan(uri, labels);
+    var span = agent.startSpan(options.hostname || options.host || 'localhost', labels);
```

Comment:
A simple comment here would be good
// c.f. _http_client.js ClientRequest constructor